ci-operator/step-registry/release-qe-tests/release-qe-tests-commands.sh

```diff
@@ -16,7 +16,7 @@ export RELEASE_PAYLOAD_MODIFIER_TOKEN=$release_payload_modifier_token
 export GCS_CRED_FILE=/var/run/vault/release-tests-sa/gcs_sa_for_qe_artifact_access.json
 
 echo "Login cluster app.ci"
```

Comment:
Shall we comment out this line, too ?

Reply:
I think we don't have to do this, when the new token is available. I will revert this change

Reply:
@liangxia this is a temp w/a, the change will be reverted when the token issue is fixed.